src/components/CommentList/CommentList.tsx

```diff
@@ -1,5 +1,19 @@
 import React from 'react';
+import { CommentFromServer } from '../../types/CommentFromServer';
+import { CommentInfo } from '../CommentInfo';
 
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+import './CommentList.scss';
+
+type Props = {
+  comments: CommentFromServer[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <>
+    <div className="CommentList">
+      {comments.map((comment) => (
+        <CommentInfo comment={comment} key={comment.id} />
+      ))}
+    </div>
+  </>
```

Comment:
```suggestion
    <div className="CommentList">
      {comments.map((comment) => (
        <CommentInfo comment={comment} key={comment.id} />
      ))}
    </div>
```
Don't use React fragment, if you already have a wrapper